src/components/Albums.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react'
+// import { Link } from 'react-router-dom';
+import { Jumbotron, Container, Row, Col, Image, Button } from 'react-bootstrap';
+import './Albums.css';
+
+
+
+export default class Albums extends Component {
+    render() {
+        return (
```

Comment:
Put all data to some array and then use `.map` over it. It will improve readability and flexability of the code.